core/src/test/java/org/apache/iceberg/TestMergeAppend.java

```diff
@@ -92,28 +83,28 @@ public void testEmptyTableAppend() {
         statuses(Status.ADDED, Status.ADDED));
   }
 
-  @Test
+  @TestTemplate
   public void testEmptyTableAppendManifest() throws IOException {
-    Assert.assertEquals("Table should start empty", 0, listManifestFiles().size());
+    assertThat(listManifestFiles()).isEmpty();
 
     TableMetadata base = readMetadata();
-    Assert.assertNull("Should not have a current snapshot", base.currentSnapshot());
-    Assert.assertEquals("Last sequence number should be 0", 0, base.lastSequenceNumber());
+    assertThat(base.currentSnapshot()).isNull();
+    assertThat(base.lastSequenceNumber()).isEqualTo(0);
 
     ManifestFile manifest = writeManifest(FILE_A, FILE_B);
     Snapshot committedSnapshot = commit(table, table.newAppend().appendManifest(manifest), branch);
 
-    Assert.assertNotNull("Should create a snapshot", committedSnapshot);
+    assertThat(committedSnapshot).isNotNull();
     V1Assert.assertEquals(
         "Last sequence number should be 0", 0, table.ops().current().lastSequenceNumber());
     V2Assert.assertEquals(
         "Last sequence number should be 1", 1, table.ops().current().lastSequenceNumber());
     List<ManifestFile> manifests = committedSnapshot.allManifests(table.io());
     ManifestFile committedManifest = Iterables.getOnlyElement(manifests);
     if (formatVersion == 1) {
-      Assertions.assertThat(committedManifest.path()).isNotEqualTo(manifest.path());
+      assertThat(committedManifest.path()).isNotEqualTo(manifest.path());
```

Comment:
in one of the other classes in this PR you switched the actual/expected parts

Reply:
Oh sorry for this. Let me check if the other part should be reverted or not.

Reply:
@nastra Sorry, I just remove `Assertions` here because this part (and other parts) are already the AssertJ style. Can I confirm if I should switch between actual/expected?

Reply:
I got the situation. I found that the similar tests in `TestMergeAppend` is different from `TestFastAppend`. Let me check them. Thank you.